packages/turf-helpers/index.js

```diff
@@ -248,6 +248,7 @@ function multiPolygon(coordinates, properties) {
  */
 function geometryCollection(geometries, properties) {
     if (!geometries) throw new Error('geometries is required');
```

Comment:
Added tests for this case: https://github.com/Turfjs/turf/pull/866/commits/c1edd39658d1f2dae611eb6106a4cb609aa9ab8d